letsencrypt/client/tests/revoker_test.py

```diff
@@ -0,0 +1,390 @@
+"""Test :mod:`letsencrypt.client.revoker`."""
```

Comment:
There hasn't been much precedence or consistency.  I agree that they should all be changed to something consistent.  Since we have been trying to use sphinx documentation style to reference everything (more so in recent revisions), I believe it makes sense to start converting everything.  For the sake of your sanity I will adopt your method for now and I will worry about the style of the project in general later when I actually run tests and compare the documentation.  I am much more focused on functionality right now.

https://github.com/letsencrypt/lets-encrypt-preview/blob/revoker/letsencrypt/client/tests/challenge_util_test.py#L1

https://github.com/letsencrypt/lets-encrypt-preview/blob/revoker/letsencrypt/client/tests/recovery_token_test.py#L1

https://github.com/letsencrypt/lets-encrypt-preview/blob/revoker/letsencrypt/client/tests/acme_util.py#L1

https://github.com/letsencrypt/lets-encrypt-preview/blob/revoker/letsencrypt/client/tests/client_test.py#L1

https://github.com/letsencrypt/lets-encrypt-preview/blob/revoker/letsencrypt/client/tests/client_authenticator_test.py#L1